packages/cactus-plugin-ledger-connector-daml/src/main/typescript/web-services/query-raw-iou-endpoint.ts

```diff
@@ -0,0 +1,118 @@
+import { Express, Request, Response } from "express";
```

Comment:
@raynatopedrajeta Please use `import type` here instead of regular `import`  that eliminates the need for us actually pulling in express as a production dependency of the plugin package. This applies throughout the codebase so please refactor it everywhere else as well.